docker-for-mac/index.md

```diff
@@ -249,6 +249,11 @@ experience conflicts, remove `/usr/local/bin/kubectl`.
   For more about using the Kubernetes integration with Docker Desktop, see
   [Deploy on Kubernetes](kubernetes.md){:target="_blank" rel="noopener" class="_"}.
 
+For information about the reset options, see [Logs and Troubleshooting](troubleshoot.md).
+
+> Upgrade Kubernetes
+>
+> Docker Desktop does not upgrade your cluster automatically after a new update. In order to upgrade your Kubernetes cluster to the latest version you will have to reset the **Kubernetes Cluster**, in the **Troubleshoot** menu.
```

Comment:
@lorenrh Does it have to be the Troubleshoot menu? There's also an option to reset the Kube cluster in **Preferences** > **Kubernetes** section

Reply:
It's from both. I'll add it